client/pages/library.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import NavBar from '../components/nav-bar';
+
+const url = 'https://api.rawg.io/api/games?key=2b1c564fe61d448fa953aa160239a72f&page_size=688028';
```

Comment:
For the API `key` this should be moved to the `.env` file, then you would do this in your code:

```JAVASCRIPT
const url = `https://api.rawg.io/api/games?key=${process.env.GAME_API_KEY}&page_size=688028`;
```

In your `.env` file you would do:
```
GAME_API_KEY=2b1c564fe61d448fa953aa160239a72f
```

Then finally you need to update your `webpack.config.js` file by requiring in `webpack` and adding the `plugins` section, below is a full example:
```JAVASCRIPT
require('dotenv/config');
const path = require('path');
const webpack = require('webpack');

const clientPath = path.join(__dirname, 'client');
const serverPublicPath = path.join(__dirname, 'server/public');

module.exports = {
  resolve: {
    extensions: ['.js', '.jsx']
  },
  entry: clientPath,
  output: {
    path: serverPublicPath
  },
  module: {
    rules: [
      {
        test: /\.jsx?$/,
        include: clientPath,
        use: {
          loader: 'babel-loader',
          options: {
            plugins: [
              '@babel/plugin-transform-react-jsx'
            ]
          }
        }
      }
    ]
  },
  plugins: [
    new webpack.DefinePlugin({
      'process.env': JSON.stringify(process.env)
    })
  ],
  devtool: 'source-map',
  devServer: {
    host: '0.0.0.0',
    port: process.env.DEV_SERVER_PORT,
    static: {
      directory: serverPublicPath,
      publicPath: '/',
      watch: true
    },
    proxy: {
      '/api': `http://localhost:${process.env.PORT}`
    }
  },
  stats: 'summary',
  performance: {
    hints: false
  }
};
```